prototypes/styletransfer/tests/test_model.py

```diff
@@ -12,10 +12,22 @@ def _sample_img(size):
 def test_model():
     content_img = _sample_img(512)
     style_img = _sample_img(512)
-
+    init_img = _sample_img(512)
+   
     model = StyleTransfer()
 
-    content_rep, style_rep = model.feature_representations(content_img, style_img)
+    style_rep, content_rep = model.feature_representations(content_img, style_img)
+    
+    assert isinstance(content_rep, List)
```

Comment:
`List`, `Tuple` are commonly used for type annotations, use `list`, `tuple`

Reply:
also those weren't imported?

Reply:
done